utils/api.ts

```diff
@@ -55,3 +59,27 @@ export const getSHMNewsArticles = (): Promise<NewsItem[]> => {
       );
   });
 };
+
+export function getSuperShardians() {
+  configureAirtable();
+  const data: any[] = [];
+  const base = Airtable.base(process.env.NEXT_PUBLIC_AIRTABLE_BASE_ID as string);
+  return new Promise<any>((resolve, reject) => {
+    base(process.env.NEXT_PUBLIC_AIRTABLE_SUPERSHARDEUM as string)
+      .select({
+        view: "Grid view",
+      })
+      .firstPage()
```

Comment:
Please add conditions for null check in this chain

Reply:
Done